pkg/datagatherer/versionchecker/versionchecker.go

```diff
@@ -257,7 +277,29 @@ type containerResult struct {
 
 // Fetch retrieves cluster information from GKE.
 func (g *DataGatherer) Fetch() (interface{}, error) {
-	rawPods, err := g.dynamicDg.Fetch()
+	// Get nodes information to update version-checker architecture structure
+	rawNodes, err := g.nodeDynamicDg.Fetch()
+	if err != nil {
+		return nil, fmt.Errorf("failed to fetch nodes, %s", err)
```

Comment:
I would recommend using `%v` when printing error rather than `%s`, so it prints all the content of the error, which can be useful especially if the error has been wrapped multiple times.

Not a hard requirement though.